scripts/update-homebrew.sh

```diff
@@ -23,11 +23,10 @@ PATH=$PATH:$HOME/.linuxbrew/bin/
 pushd ~/.linuxbrew/Library/Taps/homebrew/homebrew-core
 #git remote set-url origin https://github.com/Daniel15/homebrew-core # for testing
 git remote set-url origin https://github.com/homebrew/homebrew-core
-git reset --hard HEAD
-git clean -fd
 git fetch --prune origin
+git reset --hard origin/master
+git clean -fd
 # Remove any existing branch (eg. if the previous attempt failed)
-git checkout master
```

Comment:
Will `git reset --hard origin/master` switch to the `master` branch? I guess that's why this is not needed?

Could you try switching to a different branch, then running this script, and ensuring it switches to master?

Reply:
Ah, my bad. Didn't realize we had to be on master.